### PR TITLE
fixes #13066 - blank slate pages should be aligned to the left

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -551,3 +551,10 @@ span.glyphicon.host-status {
   max-height: 300px;
   overflow-y: auto;
 }
+
+.blank-slate-pf {
```
Comment (tbrisker) [inline app/assets/stylesheets/application.scss]: patternfly overrides should go in to the `patternfly_and_overrides.scss` file.

**Comment by dLobatog** [comment]
This violates [empty state pattern](https://www.patternfly.org/patterns/empty-state/) in Patternfly - not sure how to address this. @kybaker mind taking a look? This is how it looks like now:

screenshot from 2016-02-05 17-21-56

**Comment by shlomizadok** [comment]
Seems good to merge :+1:

**Comment by tbrisker** [comment]
I agree with @dLobatog that while this addresses the original issue, this also does not fit the patternfly pattern.I'm not sure which version actually looks better, but tend to lean towards the UX-tested pattern of patternfly.
@unorthodoxgeek - could you please add a few before/after screenshots here for several pages?
@kybaker - I would appreciate your input on the matter.

**Comment by tbrisker** [comment]
Also - apparently not all the empty state pages were changed to the new layout, which causes inconsistency in the UI.

**Comment by kybaker** [comment]
@dLobatog @tbrisker It would be great if we could start using the blank state pattern. Ideally the blank slate would be used throughout the UI but we have to start somewhere. I think you can use the same icon as well. It is already coded as well.

https://www.patternfly.org/patterns/empty-state/

**Comment by kybaker** [comment]
@tbrisker - Try making the width smaller and add some padding above. The issue is the content is not a single paragraph. I scaled my browser and it looks okay. It is not ideal but it is better than abandoning the pattern and being inconsistent with the rest of the product portfolio. Also the gray background may help frame the content better.

I doubled the content here and it still scales well as a single paragraph: 
https://dl.dropboxusercontent.com/u/5892944/Screen%20Shot%202016-03-02%20at%2011.25.02%20AM.png

https://dl.dropboxusercontent.com/u/5892944/Screen%20Shot%202016-03-02%20at%2011.18.15%20AM.png

**Comment by tbrisker** [comment]
@unorthodoxgeek looks from @kybaker's comment that the way forward is to make the text narrower - please try to do so.

**Comment by unorthodoxgeek** [comment]
@tbrisker - done

**Comment by unorthodoxgeek** [comment]
[test]

**Comment by unorthodoxgeek** [comment]
> 1 failing check
> Details default — Job result: FAILURE
> This branch has no conflicts with the base branch
> Only those with write access to this repository can merge pull requests.

**Comment by unorthodoxgeek** [comment]
תומר, תעשה לי מרג'!
@tbrisker

File: app/assets/stylesheets/patternfly_and_overrides.scss [hunk 2]
```diff
@@ -292,3 +292,9 @@ div.form-group .pficon-info{
 a.btn-info span{
   padding-bottom: 4px;
 }
+
+.blank-slate-pf {
+  font-size: 1.4em;
+  width: 70%;
```
Comment (tbrisker) [inline app/assets/stylesheets/patternfly_and_overrides.scss]: Look like 750px works better, 70% can be too much for wide screens.
But please also add a `max-width:90%` as well to make sure it is still readable even on mobile.

Comment (unorthodoxgeek) [inline app/assets/stylesheets/patternfly_and_overrides.scss]: done

**Comment by tbrisker** [comment]
Merged as 35ffdf5d6d961c7b9ec886e028fa23d6da629050, thanks for all the fish @unorthodoxgeek!!

**Pull request closed without merging**